src/VsixV3/Microsoft.Net.Sdk.swr

```diff
@@ -0,0 +1,32 @@
+use vs
+
+package name=Microsoft.Net.Sdk
+        version=$(Version)
+
+vs.localizedResources
+  vs.localizedResource language=en-us
+                       title="Microsoft Net Sdk"
+                       description="SDK for .NET projects"
+
+folder "InstallDir:MSBuild\Sdks\Microsoft.Net.Sdk\sdk"
```

Comment:
I think we should try to get the casing right in any case.  For now that means Microsoft.NET.Sdk\Sdk\Sdk.props, I believe.